From 9fec5350a0260dea619cfc4ac7bbb90003ef180b Mon Sep 17 00:00:00 2001
From: Dave Pacheco <dap@joyent.com>
Date: Thu, 21 Jul 2016 17:41:16 -0700
Subject: [PATCH] TOOLS-1494 crrestore should pull latest images Reviewed by:
 Joshua M. Clulow <jmc@joyent.com> Approved by: Alex Wilson
 <alex.wilson@joyent.com>

---
 bin/crrestore | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/bin/crrestore b/bin/crrestore
index a933acf..9a24958 100755
--- a/bin/crrestore
+++ b/bin/crrestore
@@ -118,6 +118,12 @@ function main
 	gr_backup_path="$1"
 	gr_configure || fail "failed to configure"
 
+	echo "Pulling latest copies of cr.joyent.us images ... "
+	if ! docker pull $gr_image_appserver ||
+	   ! docker pull $gr_image_frontdoor; then
+	   	fail "failed"
+	fi
+
 	echo -n "Creating database volume container ... "
 	docker run --name=$gr_name_volume_db \
 	    -v /gerrit-postgres-db-data $gr_image_volumes \
-- 
2.21.0

